aries_vcx/src/handlers/connection/public_agent.rs

```diff
@@ -31,11 +32,11 @@ impl PublicAgent {
     ) -> VcxResult<Self> {
         let pairwise_info = PairwiseInfo::create(wallet_handle).await?;
         let agent_info = CloudAgentInfo::create(agency_client, &pairwise_info).await?;
-        let service = AriesService::create()
+        let service = EndpointService::create()
```

Comment:
This would be breaking change; Is there a reason you want to do this modification, or would you mind resetting changes in this file?

Reply:
I don't understand what your point was here?

Did you mean that this change AriesService to EnpointService is incorrect?


Or should I generate a new public_invite_endpoint class, and roll back the changes to this class?

Reply:
I would like `PublicAgent` to construct `AriesService` like before, and write on ledger using the legacy method. Basically reset changes in this file, unless you have your own application reason to update PublicAgent to the new approach - if yes, we can discuss solution. 

Side note: We are planning to remove PublicAgent altogether in short-medium term unless there's interest from community to preserve it.